python/ray/tune/trainable.py

```diff
@@ -1021,3 +1022,30 @@ def build_config(self, config: Dict):
         new_config[STDOUT_FILE] = self._stdout_file
         new_config[STDERR_FILE] = self._stderr_file
         return new_config
+
+
+@PublicAPI(stability="alpha")
+class ConvertibleToTrainable(abc.ABC):
+    """A class that can be converted to a ``tune.Trainable``."""
+
+    @abc.abstractmethod
+    def as_trainable(self) -> Type[Trainable]:
+        """Convert self to a ``tune.Trainable`` class."""
+        raise NotImplementedError
+
+    def _override_attributes_with_config(self, config: Dict) -> Dict:
+        """Overrides attributes of self with values in ``config``.
```

Comment:
Actually, this can be a util function right? It doesn't need to be in the interface definition. Can probably just remove it from this PR.

Reply:
ok yeah sure I will remove this.